### PR TITLE
Updated breadcumb navigation

### PR DESCRIPTION
**Pull request by adamzemmoura**
Added breadcrumb trail to left hand side.
Increased font-size to 20px.
Changed icon from fa-dashboard to fa-home.
Updated each page so that correct name appears in breadcrumb trail.
Home link and current page links now work as expected. 
Added custom_css.css file to apply some custom styling without changing bootstrap file.

**NOTE: Needs updated to handle multi-layer navigation i.e. any links between home and current page if won't currently work**

### CODE REVIEW

File: public/css/custom-css.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*Style the seperator in the breadcrumb navigation*/
+.breadcrumb > li + li:before {
+    color: #DB4C3F;
+    content: ">";
+    padding: 0 5px;
+}
```
Comment (jmandrade) [inline public/css/custom-css.css]: Trailing newline is needed.

**Comment by jmandrade** [comment]
So, you only made changes to the CSS, it seems. We need to remove the breacrumbs from master.blade.php and they need to be added to each page.

Did you modify the minified bootstrap.min.css file? It's listed under changes, but it's not possible to do code review on a minified file.

File: public/css/custom-css.css (hunk 1 above)
Comment (adamzemmoura) [reply public/css/custom-css.css]: Not sure what you mean by a trailing new line?  Added where?  Why?

**Comment by adamzemmoura** [comment]
I tried to update the bootstrap file but realizing I shouldnt be doing that and thought I changed it back... any ideas how I can make 100% it's the way it was? Copy / paste the file from Master?

I did change remove the breadcrumbs from master and added it to each page, think I forgot to push those changes.  They should be there now with the last commit.

File: public/css/custom-css.css (hunk 1 above)
Comment (jmandrade) [reply public/css/custom-css.css]: At the very end of the file. Each file must end on a newline or Github will complain.

**Comment by jmandrade** [comment]
Yeah. You may copy from master and replace it, to bring it back (there are other ways like "resetting", but copying is faster).

Besides that, it looks good!

File: public/css/custom-css.css (hunk 1 above)
Comment (adamzemmoura) [reply public/css/custom-css.css]: Ahh okay, thanks!

**Comment by adamzemmoura** [comment]
Copy and pasted the bootstrap.min.css file from master to make sure it's the correct one.
Added a blank line at EOF.

**Pull request merged**